tests/deposit.test.js

```diff
@@ -0,0 +1,66 @@
+import HomePage from "../pages/HomePage";
+import SignIn from "../pages/SignIn";
+import CheckingPageTable from "../pages/CheckingPageTable";
+import DepositForm from "../pages/DepositForm";
+import {accountsSavingsDeposit} from "../datos/CorrectSavingsDeposit";
+import {accountsCheckingsDeposit} from "../datos/CorrectCheckingsDeposits";
+import {accountsIncorrectDeposit} from "../datos/IncorrectDeposits";
+import SavingsTable from "../pages/SavingsTable";
+
+
+describe('Deposits', () => {
+    beforeEach(async () => {
+            await SignIn.open('bank/login');
+            let username = 'jsmith@demo.io';
+            let password = 'Demo123!';
+            await SignIn.logIn(username, password);
+    });
+    it('Should take you to Deposit form', async ()=> {
+        await HomePage.accessNewDepositForm()
+        await expect(DepositForm.pageTitle).toHaveText('Deposit')
+    })
+    it('Should show the balance when selecting Individual Checking', async ()=> {
+        await HomePage.accessNewDepositForm()
+        await DepositForm.selectDepositAccount('Individual Checking (Standard Checking)')
+        await expect(DepositForm.accountBalance).toBeDisplayed();
+    })
+    it('Should show the balance when selecting Individual Savings', async ()=> {
+        await HomePage.accessNewDepositForm()
+        await DepositForm.selectDepositAccount('Indiviudal Savings (Money Market)')
+        await expect(DepositForm.accountBalance).toBeDisplayed();
+    })
```

Comment:
I'd create an array of strings (every string representing a different type of account) and loop through it.

Reply:
I did'nt understand

Reply:
I'll contact you through Google Meet.